src/application.ts

```diff
@@ -69,9 +69,6 @@ export class PocketGatewayApplication extends BootMixin(ServiceMixin(RepositoryM
       INFLUX_URL,
       INFLUX_TOKEN,
       INFLUX_ORG,
-      AWS_ACCESS_KEY_ID,
```

Comment:
Wait, why did we lose these?  they are now required for timestream

Reply:
they're now only checked on `environment.observer.ts` they will still throw error if not added, see here: https://github.com/pokt-foundation/portal-api/blob/a60a159f1359b1a31fc9116f4a1094db0ecb7b07/src/observers/environment.observer.ts#L41